Update readme

JJMats opened the pull request:
Please take a look at the README file

orcsun commented:
thank you Jeremy, the readme is in a really good shape. Many thanks!

Some minor comment from my side:
1. for consistency, maybe:
/twist_controller/dbw_node.py  -> /twist_controller
/waypoint_follower/waypoint_follower.py -> /waypoint_follower

2. Shall we modify the title of "Known Issues" to "System Optimization" to make it sounds more positive. :)
Also do you think it's necessary to add a bit more details about how your improved the braking system in this section?

3. Does anyone have a better idea for the team name? :)

Yang

monsieurmona commented:
My Family name is: Lüder. If you omit the dots on the u, the word becomes "Bitch" :-)

JJMats commented:
Ok, I made the updates as you all have suggested. Thanks for the feedback here.

I removed the "Known Issues" section and moved that content up to the Traffic Light Detector section, since it refers to a function in that node.
